### ISSUE TITLE
cuckoo is not able to process the virus file named in chinese

### ISSUE THREAD

**Issue by ccwenwen** [cuckoosandbox/cuckoo#950]
cuckoo will report error if I submit the virus file  named  in Chinese, the printed log attached below:

2016-06-08 17:18:49,743 [lib.cuckoo.core.scheduler] INFO: Starting analysis of FILE "安全保卫.exe" (task #7, options "")
2016-06-08 17:18:49,876 [lib.cuckoo.core.scheduler] INFO: Task #7: acquired machine cuckoo1 (label=cuckoo1)
2016-06-08 17:18:49,907 [modules.auxiliary.sniffer] INFO: Started sniffer with PID 5943 (interface=vboxnet0, host=192.168.56.101, pcap=/opt/cuckoo/storage/analyses/7/dump.pcap)
tcpdump: listening on vboxnet0, link-type EN10MB (Ethernet), capture size 65535 bytes
2016-06-08 17:19:05,015 [lib.cuckoo.core.scheduler] ERROR: Analysis failed: The package "modules.packages.exe" start function encountered an unhandled exception: 'ascii' codec can't encode characters in position 54-56: ordinal not in range(128)

**Comment by jbremer** [comment]
Which version of Cuckoo are you running? I'm actually getting a different fatal error.
I'm guessing that a unicode error accidentally got re-introduced again, as this used to work.
Will have to test again later, thanks for reporting.

```
Traceback (most recent call last):
  File "C:\tmpgolq94\analyzer.py", line 766, in <module>
    success = analyzer.run()
  File "C:\tmpgolq94\analyzer.py", line 572, in run
    self.target = self.package.move_curdir(self.target)
  File "C:\tmpgolq94\lib\common\abstracts.py", line 105, in move_curdir
    os.rename(filepath, outpath)
WindowsError: [Error 2] The system cannot find the file specified
```

**Comment by ccwenwen** [comment]
I tested for the problem again and got the log you want.please find the printed log:

2016-06-12 18:23:58,104 [lib.cuckoo.core.scheduler] INFO: Starting analysis of FILE "工作会议.exe" (task #12, options "")
2016-06-12 18:23:58,154 [lib.cuckoo.core.scheduler] INFO: File already exists at "/opt/cuckoo/storage/binaries/53f3201e55a11dce9a948f4326c375d6a29f5ea3d1f6faef4048da6c900271ea"
2016-06-12 18:23:58,235 [lib.cuckoo.core.scheduler] INFO: Task #12: acquired machine cuckoo1 (label=cuckoo1)
2016-06-12 18:23:58,249 [modules.auxiliary.sniffer] INFO: Started sniffer with PID 4726 (interface=vboxnet0, host=192.168.56.101, pcap=/opt/cuckoo/storage/analyses/12/dump.pcap)
tcpdump: listening on vboxnet0, link-type EN10MB (Ethernet), capture size 65535 bytes
2016-06-12 18:24:02,980 [lib.cuckoo.core.guest] INFO: Starting analysis on guest (id=cuckoo1, ip=192.168.56.101)
2016-06-12 18:24:13,130 [lib.cuckoo.core.scheduler] ERROR: Analysis failed: The package "modules.packages.exe" start function encountered an unhandled exception: 'ascii' codec can't encode characters in position 54-56: ordinal not in range(128)
8 packets captured
8 packets received by filter
0 packets dropped by kernel
2016-06-12 18:24:19,784 [lib.cuckoo.core.scheduler] INFO: Task #12: reports generation completed (path=/opt/cuckoo/storage/analyses/12)
2016-06-12 18:24:19,881 [lib.cuckoo.core.scheduler] INFO: Task #12: analysis procedure completed

the analysis.log attached below(the time is not synchronized):
2016-06-13 02:23:57,019 [analyzer] DEBUG: Starting analyzer from: C:\cjipfbu
2016-06-13 02:23:57,019 [analyzer] DEBUG: Pipe server name: \\.\PIPE\uzUOeaWNXUDgrAJitCZYdqfJuqXWyJT
2016-06-13 02:23:57,019 [analyzer] DEBUG: Log pipe server name: \\.\PIPE\kKJPAkvQPUxhHAwrlOtRLXusHPYB
2016-06-13 02:23:57,019 [analyzer] DEBUG: No analysis package specified, trying to detect it automagically.
2016-06-13 02:23:57,019 [analyzer] INFO: Automatically selected analysis package "exe"
2016-06-13 02:24:00,073 [analyzer] DEBUG: Started auxiliary module Disguise
2016-06-13 02:24:00,273 [analyzer] DEBUG: Loaded monitor into process with pid 644
2016-06-13 02:24:00,273 [analyzer] DEBUG: Started auxiliary module DumpTLSMasterSecrets
2016-06-13 02:24:00,293 [analyzer] DEBUG: Started auxiliary module Human
2016-06-13 02:24:00,293 [analyzer] DEBUG: Started auxiliary module InstallCertificate
2016-06-13 02:24:00,293 [analyzer] DEBUG: Started auxiliary module Screenshots

**Comment by jbremer** [comment]
I believe this issue has been resolved as part of #863 as I tested this a few times recently without running into any exceptions. Closing as resolved.
